Automattic/wp-calypso
# Stats: Removing released feature flag

Related to #73674

## Proposed Changes

* removing feature flag for the Insights page grid that was released few weeks ago

## Testing Instructions



* navigate to the `Insights` page on the live branch
* verify that the grid loads properly and all modules look good
* test across different breakpoints

## Pre-merge Checklist
